code/modules/clothing/suits/ego_gear/waw.dm

```diff
@@ -1,10 +1,10 @@
-// WAW Armor should be kept at ~135 total armor.
+// WAW Armor should be kept at ~130 total armor.
```

Comment:
Make it 140 instead, if you really need it.

Reply:
Done.